_data/sites.json

```diff
@@ -9478,6 +9478,15 @@
             "shopify.com"
         ]
     },
+    {
+        "name": "Shop Your Way",
```

Comment:
According to the CI:
```
Keys must be in alphanumeric order. shopify needs to come after shop your way
```

I suspect due to the whitespace it will also need to come before `Shopee`

Reply:
Indeed it does. I've updated the order. I ran the "validate_json.rb" script this time around and it didn't show any errors.